### ISSUE TITLE
kubectl not finding kubelogin from Path VAR

### ISSUE THREAD

**Issue by kevball2** [Azure/kubelogin#313]
working on a windows 11 VM. Manually installed kubectl and kubelogin and created entries in both user and machine PATH variables. both kubelogin and kubectl can be found individually. When I call kubectl version though, it cannot find kubelogin.  I have tried multiple versions of kubectl and kubelogin. Pretty sure this a environment variable issue but I can't see what I am missing. 

```
kubectl version
WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.6", GitCommit:"ff2c119726cc1f8926fb0585c74b25921e866a28", GitTreeState:"clean", BuildDate:"2023-01-18T19:22:09Z", GoVersion:"go1.19.5", Compiler:"gc", Platform:"windows/amd64"}
Kustomize Version: v4.5.7
Unable to connect to the server: getting credentials: exec: executable kubelogin not found

It looks like you are trying to use a client-go credential plugin that is not installed.

To learn more about this feature, consult the documentation available at:
      https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins


kubelogin is not installed which is required to connect to AAD enabled cluster.

To learn more, please go to https://aka.ms/aks/kubelogin
```

trying to connect to run commands to check a cluster results in the same error. 
```
kubectl get pods
Unable to connect to the server: getting credentials: exec: executable kubelogin not found

It looks like you are trying to use a client-go credential plugin that is not installed.

To learn more about this feature, consult the documentation available at:
      https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins


kubelogin is not installed which is required to connect to AAD enabled cluster.
```

kubelogin does convert the config properly 

```
kubelogin convert-kubeconfig -l azurecli -v=7
I0807 11:26:48.689605    4556 convert.go:160] Context: , Login Method: azurecli, Environment: AzurePublicCloud, TenantID: , ServerID: , ClientID: , IsLegacy: false, msiResourceID: , tokenCacheDir: C:\Users\<user>/.kube/cache/kubelogin/, tokenCacheFile: C:\Users\<user>\.kube\cache\kubelogin\AzurePublicCloud---.json, AZURE_CONFIG_DIR:
I0807 11:26:48.690854    4556 convert.go:170] Loading kubeconfig from C:\Users\<user>\.kube\config
I0807 11:26:48.692892    4556 loader.go:373] Config loaded from file:  C:\Users\<user>\.kube\config
I0807 11:26:48.693418    4556 convert.go:192] context: "<clusterName>"
I0807 11:26:48.693578    4556 convert.go:199] converting...
I0807 11:26:48.695403    4556 loader.go:373] Config loaded from file:  C:\Users\<user>\.kube\config
I0807 11:26:48.696269    4556 loader.go:373] Config loaded from file:  C:\Users\<user>\.kube\config
```

Path variables for reference 
```
$env:Path
C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Users\adminuser\.azure-kubectl;C:\Users\<user>\.azure-kubelogin;C:\Program Files\Docker\Docker\resources\bin;C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Users\<user>\.azure-kubectl;C:\Users\<user>\.azure-kubelogin;C:\Program Files\Docker\Docker\resources\bin;
```

**Comment by weinong** [comment]
can you try to specify `kubelogin.exe` in your kubeconfig? i think the default is just `kubelogin`

**Comment by kevball2** [comment]
@weinong, 
I was able to work around the issue by navigating into the `C:\Users\<user>\.azure-kubelogin` folder and I was able to run all the commands successfully. Unfortunately, the VM I was using was deleted so I cannot test your suggestion. going to this off as a weird pathing issue. 

tl:dr
Issue: Unable to run kubekctl commands against aks, received errors that kube-login was not found on the system even though it was installed and could be reached by manually calling the exe. 
Resolution: Navigated into the `C:\Users\<user>\.azure-kubelogin` folder and kubectl was able to find the kubelogin.exe successfully.
